torch/functional.py

```diff
@@ -755,9 +761,10 @@ def stft(input: Tensor, n_fft: int, hop_length: Optional[int] = None,
 def _unique_impl(input: Tensor, sorted: bool = True,
                  return_inverse: bool = False, return_counts: bool = False,
                  dim: Optional[int] = None) -> _unique_impl_out:
-    r"""unique(input, sorted=True, return_inverse=False, return_counts=False, dim=None) -> Tuple[Tensor, Tensor, Tensor]
```

Comment:
Same as above: we can leave this as is and add a comment: # noqa D400 - which should skip this line.